dll/win32/shell32/CMakeLists.txt

```diff
@@ -114,7 +114,7 @@ set_source_files_properties(shell32.rc PROPERTIES OBJECT_DEPENDS ${CMAKE_CURRENT
 set_module_type(shell32 win32dll UNICODE)
 target_link_libraries(shell32 shellmenu shelldesktop wine uuid recyclebin cpprt atl_classes)
 add_delay_importlibs(shell32 powrprof shdocvw devmgr winspool.drv winmm mpr uxtheme ole32 oleaut32 userenv browseui version fmifs)
-add_importlibs(shell32 advapi32 gdi32 user32 comctl32 comdlg32 shlwapi msvcrt kernel32 ntdll)
+add_importlibs(shell32 advapi32 gdi32 user32 comctl32 comdlg32 shlwapi msvcrt kernel32 ntdll psapi)
```

Comment:
I checked with MSDepends: Win 2k3sp2 shell32.dll does not directly import anything from psapi.dll at all. Neither static nor delay-loaded import. A perfect solution would be avoiding that import altogether therefore.
In case you wouldn't be able to avoid it entirely in the final solution, then a delay-load is the weaker form of violation.

Reply:
Indeed, Windows' shell32 doesn't require enumerating processes in order to find the opened control panels windows.

Reply:
This is already a moot point based on my comments?